pkg/warmup/target.go

```diff
@@ -64,7 +64,7 @@ func (t Target) WaitForReadinessProbe() error {
 	for {
 		select {
 		case <-timeout:
-			return fmt.Errorf("Giving up! Target not ready after %d seconds 🙁", t.options.ReadinessTimeoutInSeconds)
```

Comment:
Out of curiosity, is that a convention in GO to avoid starting any messages with a capital letter?

Reply:
This is just for errors strings because they are printed following other context.

For example `log.Printf("Error opening file %s: %v", filename, err)`

Reply:
Yep: https://github.com/golang/go/wiki/CodeReviewComments#error-strings